benchmarks/squad/squad.py

```diff
@@ -0,0 +1,237 @@
+import re
+import string
+
+from typing import List, Tuple, Union
+
+import numpy as np
+from scipy.optimize import linear_sum_assignment  # type: ignore
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.scorer import Score, Target, bootstrap_std, mean, scorer
+from inspect_ai.solver import system_message, generate, TaskState
+
+
+SYSTEM_MESSAGE = """
+Using only the context provided in the question itself, you are tasked with answering a question in as few words as possible.
+"""
+
+@task
+def squad():
+   
+    dataset = hf_dataset(
+        path="rajpurkar/squad",
+        split="validation",
+        sample_fields=record_to_sample,
+        trust=True,
+        shuffle=True
+    )
+
+    return Task(
+        dataset=dataset,
+        plan=[
+            system_message(SYSTEM_MESSAGE),
+            generate()
+        ],
+        scorer=squad_f1_scorer()
+    )
+
+def record_to_sample(record):
+  return Sample(
+      input=format_input(record),
+      target=format_target(record),
```

Comment:
```suggestion
      target=record["answers"]["text"],
```

No need to create a separate function for this.